CMakeLists.txt

```diff
@@ -108,11 +108,11 @@ target_compile_definitions(${PROJECT_NAME} PRIVATE
 )
 
 # if INCLUDE_MIDI
-target_link_libraries(${PROJECT_NAME} 
+target_link_libraries(${PROJECT_NAME}
     tinyusb_device
     tinyusb_board
 )
-pico_enable_stdio_usb(${PROJECT_NAME} 0)
+pico_enable_stdio_usb(${PROJECT_NAME} 1)
```

Comment:
had to pass this to 1 for the `printf` calls to print to the USB (`comms -serial`)